src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -1203,6 +1203,28 @@
             </div>
           </div>
         </ng-container>
+
+      <!-- RGW QAT Compression -->
+        @if(serviceForm.controls.service_type.value === 'rgw') {
+        <div class="form-item"
+             formGroupName="qat">
+            @for(property of rgwQatList; track property) {
+            @if(property === 'compression') {
+          <cds-select formControlName="{{property}}"
+                      name="{{property}}"
+                      label="QAT compression mode"
+                      id="{{property}}"
+                      helperText="QAT compression is optional. Choose Hardware or Software to enable compression, or select None to disable it."
+                      i18n-helperText>
+            <option *ngFor="let option of rgwQatOptions"
+                    [value]="option"
+                    i18n>{{ qatDisplayMap[option] }} </option>
```

Comment:
if we were to do it like I mentioned above then we can simplify this with

```suggestion
            <option *ngFor="let option of qatCompressionOptions"
                    [value]="option.value"
                    i18n>{{ option.label }} </option>
```